src/codeswitch/data_generation/generate_prompt_template.py

```diff
@@ -0,0 +1,143 @@
+        [Insert an English sentence with an object phrase that is likely to be code-switched to Spanish.]
+        [Insert a Spanish sentence with a noun phrase that is likely to be code-switched to English.]
+        [Insert a Spanish sentence with an action verb phrase that is likely to be code-switched to English.]
+        [Insert a Spanish sentence with an object phrase that is likely to be code-switched to English.]
+        """,
+        "Technical Support or IT Assistance": """
+        [Technical Support or IT Assistance]:
+        - Use technical terms appropriately and provide clear troubleshooting steps.
+        - Maintain a patient and supportive tone to assist users with varying technical skills.
+        [Insert an English sentence with a noun phrase that is likely to be code-switched to Spanish.]
+        [Insert an English sentence with an action verb phrase that is likely to be code-switched to Spanish.]
+        [Insert an English sentence with an object phrase that is likely to be code-switched to Spanish.]
+        [Insert a Spanish sentence with a noun phrase that is likely to be code-switched to English.]
+        [Insert a Spanish sentence with an action verb phrase that is likely to be code-switched to English.]
+        [Insert a Spanish sentence with an object phrase that is likely to be code-switched to English.]
+        """,
+        "Transportation or Ride-Sharing Services": """
+        [Transportation or Ride-Sharing Services]:
+        - Use polite and professional language to ensure a positive interaction.
+        - Include terms related to transportation logistics and passenger communication.
+        [Insert an English sentence with a noun phrase that is likely to be code-switched to Spanish.]
+        [Insert an English sentence with an action verb phrase that is likely to be code-switched to Spanish.]
+        [Insert an English sentence with an object phrase that is likely to be code-switched to Spanish.]
+        [Insert a Spanish sentence with a noun phrase that is likely to be code-switched to English.]
+        [Insert a Spanish sentence with an action verb phrase that is likely to be code-switched to English.]
+        [Insert a Spanish sentence with an object phrase that is likely to be code-switched to English.]
+        """,
+    }
+
+    return templates.get(context, "")
+
+
+# Generate code-switched prompts for different contexts
+CONTEXTS = [
+    "Business Meeting",
+    "Helpdesk Communication",
+    "Contact Center Communication",
+    "Customer Service/Support",
+    "Logistics or Delivery Coordination",
+    "Technical Support or IT Assistance",
+    "Transportation or Ride-Sharing Services",
+]
+
+
+def main():
+    num_instances = 10  # Change this to the desired number of instances
+    prompts = generate_code_switched_prompts(CONTEXTS, num_instances)
+
+    # Create DataFrame
+    df = pd.DataFrame({"Prompt": [prompt[1] for prompt in prompts]})
```

Comment:
If you're only using the index 1 item, why are you appending something else to the prompt output at line  [23](https://github.com/cheyannelam/code-switch/pull/2/files#diff-55a88ac593e2913f3f00f40f297b407fa5edc461662c261e7c8d5f140ad1bdcaR23) at the first place?